### ISSUE TITLE
New RHEL installs failing

### ISSUE THREAD

**Issue by ronivay** [ronivay/XenOrchestraInstallerUpdater#256]
New installations to RHEL based distros fail currently due to forensics.cert.org repository not working.

```
# curl -I -L https://forensics.cert.org
curl: (56) OpenSSL SSL_read: Connection reset by peer, errno 104
```

As this might be a transient issue due to invalid certificate config, won't do any workaround just yet. If this isn't fixed soon, will look for an alternative.

**Comment by awstaskuser** [comment]
This might be a transient issue with their SSL configuration.

**Comment by tig987** [comment]
It's an DNS issue.
forensics.cert.org is not reachable because there are no DNS records for this domain - see
https://www.nslookup.io/domains/forensics.cert.org/dns-records/

**Comment by ronivay** [comment]
> It's an DNS issue.
> 
> forensics.cert.org is not reachable because there are no DNS records for this domain - see
> 
> https://www.nslookup.io/domains/forensics.cert.org/dns-records/

Yep now they've removed the whole DNS record. Probably won't come back then. Need to start thinking of an alternative.

**Comment by tig987** [comment]
found this blog post:
https://ludditus.com/2024/07/25/what-you-need-to-know-when-using-my-custom-almalinux-kde-iso/#comment-103637

Looks like CERT Forensics Tools will definitely not come back.

**Comment by ronivay** [comment]
Workaround is now in place to not install forensics repo or libvhdi-tools for RHEL distros. Obviously this isn't a proper fix but will allow installations to proceed until there's another working source for this package.

**Comment by ronivay** [comment]
`libvhdi-tools` pkg is still not available in any generic RHEL repository. For now only some Fedora repos seem to have it, which isn't really what i'd like to use. If no sane alternative comes up soon, i might leave it like this. If one wants to do file level restore from backups with new installs, go with Debian or Ubuntu for that.

**Comment by bnerickson** [comment]
Hi @ronivay , I've setup the following Copr with the latest libvhdi+libvhdi-tools for el8 and el9 (though tbh I've only tested el9 because that's what I use): https://copr.fedorainfracloud.org/coprs/bnerickson/libvhdi/

I've setup a notification in github for new releases in the https://github.com/libyal/libvhdi repository, so I should be keeping it up to date.  If this is all OK, I can submit a patch to update the script linking to that repo and install the libvhdi+libvhdi-tools rpms.

**Comment by ronivay** [comment]
> Hi @ronivay , I've setup the following Copr with the latest libvhdi+libvhdi-tools for el8 and el9 (though tbh I've only tested el9 because that's what I use): https://copr.fedorainfracloud.org/coprs/bnerickson/libvhdi/
> 
> I've setup a notification in github for new releases in the https://github.com/libyal/libvhdi repository, so I should be keeping it up to date.  If this is all OK, I can submit a patch to update the script linking to that repo and install the libvhdi+libvhdi-tools rpms.

Nice work. I'd appreciate a PR and i can test it with el8 too, np. Though i would make adding this repository and pkg installations specifically opt-in with a mention in the readme, that this is a small third party project.

**Comment by ronivay** [comment]
Thanks to @bnerickson this can be closed now. Optional config variable allows installation of working libvhdi-tools from the Copr repository.
